### PR TITLE
Added post_language to the extended $post variable and print it on th…

### PR DESCRIPTION
**Pull request by annalinneajohansson**
…e <article> element if it exists

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-3mm8hxy Inställningar så att talsyntesen läser upp på rätt språk](https://app.clickup.com/t/3mm8hxy)

File: views/v3/partials/article.blade.php [hunk 1]
```diff
@@ -1,4 +1,8 @@
-<article id="article" class="c-article c-article--readable-width s-article u-clearfix">
+<article id="article" class="c-article c-article--readable-width s-article u-clearfix" 
+@if($postLanguage)
```
Comment (sebastianthulin) [inline views/v3/partials/article.blade.php:2]: {!! ($postLanguage ? 'lang="'.$postLanguage.'"' : '') !!}

Comment (sebastianthulin) [reply views/v3/partials/article.blade.php:2]: Not shure if this is a better solution, but leaving a suggestion.

Comment (annalinneajohansson) [reply views/v3/partials/article.blade.php:2]: Agree, it looks better with a oneliner here. It was bothering me too that I was linebreaking the <article>. Will change.

**Pull request merged**